VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -60,14 +65,14 @@ public void showSettings(Activity sourceActivity) {
         sourceActivity.startActivity(settingsIntent);
     }
 
-    public void showLaunchScreen(Activity sourceActivity, boolean overrideAnimation) {
+    public void showLaunchScreen(Context sourceActivity, boolean overrideAnimation) {
         Intent launchIntent = new Intent(sourceActivity, LaunchActivity.class);
         launchIntent.putExtra(LaunchActivity.OVERRIDE_ANIMATION_FLAG,overrideAnimation);
         launchIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
         sourceActivity.startActivity(launchIntent);
```

Comment:
Since ``sourceActivity`` isn't an activity any more, maybe we should change the name?